mariadb-connector/src/main/java/life/qbic/authentication/persistence/ProjectJpaRepository.java

```diff
@@ -0,0 +1,44 @@
+package life.qbic.authentication.persistence;
```

Comment:
I am not sure that this is the right place to put this class.

Reply:
probably not, good point :D